common/client.go

```diff
@@ -62,6 +62,10 @@ type DatabricksClient struct {
 	AzureTenantID      string `name:"azure_tenant_id" env:"ARM_TENANT_ID" auth:"azure"`
 	AzurermEnvironment string `name:"azure_environment" env:"ARM_ENVIRONMENT"`
 
+	// Resource ID of the Azure Databricks application.
+	// This is configured automatically when an Azure authentication method is used.
+	armDatabricksResourceID string
```

Comment:
should it rather be an env variable instead?.. not to publish the uuid in the wild. this will also be easier for sdks.